modules/light-clients/06-solomachine/client_state_test.go

```diff
@@ -61,7 +62,7 @@ func (suite *SoloMachineTestSuite) TestClientStateValidate() {
 			suite.Run(tc.name, func() {
 				err := tc.clientState.Validate()
 
-				if tc.expPass {
+				if tc.expErr == nil {
```

Comment:
shouild also do a check for specific error (i.e `ErrorContains` or `ErrorIs` etc). Can take a look at other PRs to see how its done!